store/policy.go

```diff
@@ -177,6 +177,22 @@ func (s *Store) GetPipelineApprovalPolicy(ctx context.Context, environmentID int
 	return api.UnmarshalPipelineApprovalPolicy(policy.Payload)
 }
 
+// GetNormalSchemaReviewPolicy will get the normal schema review policy for an environment.
+func (s *Store) GetNormalSchemaReviewPolicy(ctx context.Context, environmentID int) (*advisor.SchemaReviewPolicy, error) {
```

Comment:
How about combine this with `GetSchemaReviewPolicyNormalByID` into one function like `GetNormalSchemaReviewPolicy(ctx context.Context, find *api.PolicyFind)`?